src/cmd/support_archive/objects_query.go

```diff
@@ -0,0 +1,81 @@
+package support_archive
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/webhook"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type objectQuery struct {
+	groupVersionKind schema.GroupVersionKind
+	listOptions      []client.ListOption
+}
+
+func getObjectQueries(namespace string) []objectQuery {
```

Comment:
This is the most maintainable form. The object queries define, which objects we are collecting. This is the input for the collecting algorithm. Here you a) have a nice overview of what we collect and b) can easily extend the list of objects by just adding a new query.

Reply:
And I wouldn't expect too much noise when collecting all objects in the operator namespace. On the contrary, if there are unexpected objects in this namespace, it is very interesting to know about them, because it is a strong indication for a faulty or at the very least non-standard operator deployment.

Reply:
I agree it's nice to collect all the queries in one place, but I don't think it applies here.

We have at least 2 different types of queries: resources in the operator namespace and other namespaces themselves. `listOptions` has 3 different values (1 of them not a list)...